mbtiles/src/mbtiles.rs

```diff
@@ -39,6 +40,81 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct ZoomStats {
+    pub zoom: u8,
+    pub count: u64,
+    pub smallest: u64,
+    pub largest: u64,
+    pub average: f64,
+    pub bbox: Bounds,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct Statistics {
+    pub file_path: String,
+    pub file_size: u64,
+    pub schema: MbtType,
+    pub page_size: u64,
+    pub zoom_stats_list: Vec<ZoomStats>,
+    pub count: u64,
+    pub smallest: Option<u64>,
+    pub largest: Option<u64>,
+    pub average: f64,
+}
+
+impl Display for Statistics {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        writeln!(f, "File: {}", self.file_path).unwrap();
+
+        let file_size = SizeFormatterBinary::new(self.file_size);
+        writeln!(f, "FileSize: {:.2}B", file_size).unwrap();
```

Comment:
inlining looks a bit better - here and below
```suggestion
        writeln!(f, "FileSize: {file_size:.2}B").unwrap();
```

Reply:
All inlined now but something like this is still [not supported](https://github.com/rust-lang/rust/issues/114108):
```rust
struct Foo { bar: i32 }

fn main() {
    let foo = Foo { bar: 10 };
    let _x = format!("Hello {foo.bar}");
}
error: invalid format string: field access isn't supported
 --> src/main.rs:5:30
  |
5 |     let _x = format!("Hello {foo.bar}");
  |                              ^^^^^^^ not supported in format string
  |
help: consider using a positional formatting argument instead
  |
5 |     let _x = format!("Hello {0}", foo.bar);
  |                              ~  +++++++++
```

Reply:
yep, only simple vars are supported at the moment